inventory/host_vars/merger01.internal.opentechsjc.bonnyci.org

```diff
@@ -0,0 +1,3 @@
+---
+zuul_components:
```

Comment:
So we discussed on IRC and decided to manage everything in host_vars, rather than a potentially confusing combo of host_vars and group_vars.